platform-dom/src/main/resources/com/softicar/platform/dom/elements/dom-modal-dialog-style.css

```diff
@@ -17,6 +17,7 @@
 	gap: var(--COLUMN_GAP);
 	min-width: 256px;
 	max-width: 512px;
+	white-space: pre-wrap;
```

Comment:
we may only do this for modal alerts, prompts and confirms - but not for modals in general.